flink-yarn/src/test/java/org/apache/flink/yarn/YarnFileStageTest.java

```diff
@@ -199,7 +199,8 @@ static void testCopyFromLocalRecursive(
 				remotePaths,
 				localResources,
 				localResourceDirectory,
-				new StringBuilder());
+				new StringBuilder(),
+				1);
```

Comment:
Why should we explicitly set `1` instead of rely on default value. I'm afraid it breaks some current user settings.

Reply:
I don't think this is going to be a problem since it is only testing a package-private functionality

Reply:
Agree.

Reply:
@walterddr what I'm concerned about is that existing logic should perfectly respect with default value, instead a configured non-default value. Otherwise it is suspected we break existing manner.

Reply:
To clarify breaking change could be ok but it should be explicit.

Reply:
I don't think that setting it to 1 is a good decision. I agree with @TisonKun 's point of view, rely on default value will be better, and it can reduce the risk of disrupting the current user settings.

Reply:
Sounds good. I updated to the default hadoop DFSConfigKeys.DFS_REPLICATION_DEFAULT, which is the default value when the YarnConfigOptions.FILE_REPLICATION is not set.